api/token.go

```diff
@@ -59,7 +59,7 @@ func (a *API) ResourceOwnerPasswordGrant(ctx context.Context, w http.ResponseWri
 	user, err := models.FindUserByEmailAndAudience(a.db, instanceID, username, aud)
 	if err != nil {
 		if models.IsNotFoundError(err) {
-			return oauthError("invalid_grant", "No user found with this email")
+			return oauthError("invalid_grant", "Invalid Password")
```

Comment:
This now matches the error message on line 72.

Reply:
@lloydjatkinson Thank you for the suggestion! If the user's email address is legitimately not found, it is probably the wrong message to say "Invalid Password". 
Ideally we combine the verbiage to create ambiguity. So the message here, AND on line 72 will be something similar to "No user found with that email, or password invalid". I hope that makes sense!

Reply:
@dustincrogers That does make sense yes, shall I update it with the message you suggest?

Reply:
Please do, friend.

Reply:
Done!